CMakeLists.txt

```diff
@@ -97,20 +101,4 @@ build_addon(inputstream.adaptive ADP DEPLIBS)
 
 include(CPack)
 
-if(NOT CMAKE_CROSSCOMPILING)
-  #Tests
-  find_package( Threads )
-  add_executable(Test_DashTree
-    src/parser/DASHTreeTest.cpp
-    src/parser/DASHTree.cpp
-    src/parser/PRProtectionParser.cpp
-    src/common/AdaptiveTree.cpp
-    src/helpers.cpp
-    src/oscompat.cpp
-    )
-  target_link_libraries(Test_DashTree ${EXPAT_LIBRARIES} Threads::Threads)
-
-  enable_testing()
-
-  add_test(6play_live Test_DashTree "${CMAKE_SOURCE_DIR}/src/parser/testfiles/6play.mpd")
-endif()
\ No newline at end of file
+add_subdirectory(src/test)
```

Comment:
this won't work for cross-compile, so it has to be
if(NOT CMAKE_CROSSCOMPILING)

Reply:
Sure thing. I saw the comment above about the need to run the compiled tests on Android/ios etc. The tests in this PR are not touching any platform specific code anyway.

Reply:
Done